### DISCUSSION TITLE
Importing texture(standard) on LookdevX Graph Editor got import error of _tf

### DISCUSSION THREAD

**Discussion by Pill-bug-net** [Autodesk/maya-usd#4057] [Q&A]
Hi guys,
I am trying to use MaterialX in Maya's LookdevX, but I have some errors when importing texture. It happens when I click the folder icon beside the filename type section on the parameters window.
This error showed up.

`# Error: ImportError: file C:\Program Files\Autodesk\Maya2025\Python\Lib\importlib\__init__.py line 126: DLL load failed while importing _tf: The specified module could not be found.
# Error: ImportError: file C:\Program Files\Autodesk\Maya2025\Python\Lib\importlib\__init__.py line 126: DLL load failed while importing _tf: The specified module could not be found.
import arnold
import mtoa
import mtoa.cmds.registerArnoldRenderer;mtoa.cmds.registerArnoldRenderer.registerArnoldRenderer()
updateRenderOverride;
import mtoa.viewport;mtoa.viewport.add_controls()
// AbcImport v1.0 using Alembic 1.8.5 (built Sep 19 2024 06:28:52)
# Error: ImportError: file C:\Program Files\Autodesk\Maya2025\Python\Lib\importlib\__init__.py line 126: DLL load failed while importing _tf: The specified module could not be found.
updateRendererUI;
+++++++ Turtle for Maya registered successfully ++++++
# Error: ImportError: file C:\Program Files\Autodesk\Maya2025\Python\Lib\importlib\__init__.py line 126: DLL load failed while importing _tf: The specified module could not be found.
# Error: ImportError: file C:\Program Files\Autodesk\Maya2025\Python\Lib\importlib\__init__.py line 126: DLL load failed while importing _tf: The specified module could not be found.
// Error: file: C:/Program Files/Autodesk/MayaUSD/Maya2025/0.30.0/mayausd/MayaUSD/plugin/adsk/scripts/mayaUsdMenu.mel line 200: Error evaluating argument at position 1 in procedure "capitalizeString".
updateRendererUI;
updateRendererUI;
file -f -new;
// untitled
# Error: ImportError: file C:\Program Files\Autodesk\Maya2025\Python\Lib\importlib\__init__.py line 126: DLL load failed while importing _tf: The specified module could not be found.`

Any advice?
Thanks

**Comment by Pill-bug-net** [comment]
I figured out what caused this problem.
I installed USD cat by myself before, and when I removed it, it seemed to be working.

**Reply by wallworm** [reply]
Thank you for the heads up and clarification.
